### ISSUE TITLE
Steam controller dongle and controller input no longer recognized by client

### ISSUE THREAD

**Issue by longshanks197** [ValveSoftware/steam-for-linux#7483]
#### Your system information

* Steam client version (build number or date): Package Version 1605919008, Built Nov 20 2020 21:55:57
* Distribution (e.g. Ubuntu): Devaun Beowulf (Debian Buster/Stable)
* Opted into Steam client beta?: Yes
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
At some point the controller input has stopped working on a previously working setup. The Steam Controller dongle and steam controllers are being recognized by the OS but not the client. Controller is recognized by the client when connected through a Steam Link device but controller input is not being sent to games. Controller has been tested with evtest and jstest with satisfactory results. Whether client is in Big Picture Mode (BPM) or not, it has no effect.

As a troubleshooting step the following have been made:

1. System reboot
2. System boot with prior kernel version (5.7.10)
3. evtest and jstest verification
4. Recovered firmware on controller -> resulted in unpairing and since the dongle is no longer identified can only pair on steam link.
5. Use old USB Gravis Gamepad -> detected by client as generic controller but no controller inputs are sent to any game

steam-devices is and has been installed 

System Messages during USB dongle insert and system information:

https://gist.github.com/longshanks197/a3d4c2f8ee567e7b020be982bad4b0e8

#### Steps for reproducing this issue:

1. In BPM or Controller settings -> No Controllers Detected
2. Unplug and plug Dongle, No change
3. Power on controller, Has desktop functions -> No Controllers Detected 
4. In game as a desktop configuration only

EDIT: fixed gist link

**Comment by longshanks197** [comment]
Just as a follow up. Rolling back to the stable branch does not fix the issue. I also can confirm that I finished a game using steam controllers on October 10, 2020. From the change logs there has been numerous changes to Steam Input even in the stable branch.

**Comment by longshanks197** [comment]
After a bit of troubleshooting I have determined that this is not a Steam Client issue. My system is Devaun Beowulf with eudev (3.2.7-6) and elogind (241.4). It seems the system was not properly honoring the uaccess tags and assigning device permissions. Work around was to get permissions to trigger on a second console login and log out of the xsession and back in and it did finally properly trigger. In my troubleshooting I did a number of changes (like switch from slim to lightdm as the display manager) but I am not confident that I have pinpointed the error. This may also be fixed in newer revisions to elogind (241.4 is basically a year old at this point) but I have not moved to the testing branch to find out. I am not sure why the issue revealed itself in the last month but regardless I am closing this issue with comments.
